.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        meteor: [1.12.2, 2.8.1, 2.13]
+        meteor: ['2.14']
```

Comment:
This should be '3.0-beta.4'

Reply:
Ok, I'm waiting an answer on https://github.com/Meteor-Community-Packages/redis-oplog/pull/13#issuecomment-1896175751.